spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
@flying-sheep Spyder, although a stand alone program by itself, is also part of a larger ecosystem of scientific computing distributions including [Anaconda](http://continuum.io/downloads), [WinPython ](https://winpython.github.io/). 

This modification may produce problems for those distributions until they have gotten things up to date. 

This change is not at the moment a need at this point because the development version is still fixing some rough edges to fully support Qt5 plus we might need to adapt the imports and some code to make the code look like Qt5 (even if bellow the curtains the user has Qt4).

I am :-1: on this one for the time being until the distributions offer complete Qt5 support.